scripts/run_e2e_tests.py

```diff
@@ -468,6 +472,49 @@ def get_chrome_driver_version():
     return chrome_driver_version
 
 
+def start_portserver():
+    """Start a portserver in a subprocess.
+
+    The portserver listens at PORTSERVER_SOCKET_PATH and allocates free
+    ports to clients. This prevents race conditions when two clients
+    request ports in quick succession. The Google Appengine we use to
+    serve the development version of Oppia uses python_portpicker, which
+    is compatible with the portserver this function starts, to request
```

Comment:
What do you mean by compatible? Perhaps elaborate.

Reply:
done in 97bba1a74